src/main/webapp/order.html

```diff
@@ -13,14 +13,22 @@
    <a href="user.html" class="button">Profile</a>
    <a href="store.html" class="button">Explore</a> 
    <a href="search.html" class="button">Search for a drink!</a> 
-    <a href="order.html" id="active" class="button">Order</a>
+   <a href="order.html" id="active" class="button">Order</a>
 </nav>
 <nav id="loginbar">
       <div id="form"></div>
       <div id="login-container"></div>
       <div id="email"></div>
 </nav>
+
+    <div>Welcome to TYD Coffee's order page! Go</div> 
+    <div>ahead and search for drinks and we'll pick our</div>
+    <div>recommended stores for you. When you're</div>
+    <div>done ordering, press submit order</div>
```

Comment:
Suggestion: highlight "submit order" differently - italics, bold, different font, etc

Reply:
makes sense, I will do that , thanks.